lib/uber-s3/operation/object/tagging.rb

```diff
@@ -0,0 +1,27 @@
+module UberS3::Operation::Object
```

Comment:
This is the pattern they used for adding options to the request. It is weird but I followed it for consistency reasons.

Reply:
yea rails mixin style